task_1_1_1__/HeapSort.java

```diff
@@ -0,0 +1,52 @@
+
+import java.util.*;
+import java.util.Arrays;
+
+public class HeapSort {
```

Comment:
Add javadocs for all public contract of the class (public fields, methods, class itself)